docs/transformers/text-normalizer.md

```diff
@@ -7,6 +7,8 @@ This transformer converts all text to lowercase and removes extra whitespace.
 
 **Data Type Compatibility:** Categorical
 
+⚠️ This transformer can't handle multibyte text properly. For multibyte text, use [MultibyteTextNormalizer](multibyte-text-normalizer.md).
```

Comment:
I like this warning ... can we format it like this to stay consistent with the rest of the docs?

> **Note:** ⚠️ This transformer can't handle multibyte text properly. For multibyte text, use [MultibyteTextNormalizer](multibyte-text-normalizer.md).

Example

https://github.com/RubixML/RubixML/edit/master/docs/classifiers/adaboost.md